docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -1586,6 +1586,94 @@ public class SecurityEventListener {
 
 TIP: You can listen to other security events as described in the xref:security-customization.adoc#observe-security-events[Observe security events] section of the Security Tips and Tricks guide.
 
+
+[[oidc-token-revocation]]
+=== Token revocation
+
+Sometimes, you may want to revoke the current authorization code flow access and/or refresh tokens.
+For example, if the user has logged out, your logout event listener can revoke access refresh tokens associated with the currenty session.
+
+`quarkus.oidc.OidcProviderClient` which provides access to the OIDC provider's UserInfo, token introspection and revocation endpoints, can be used to support such cases.
```

Comment:
It reads bit strange to me to start with code `quarkus.oidc.OidcProviderClient`. Not sure if `The` belongs there, but let see if anyone else things that too. If not, it's OK.

Reply:
@michalvavrik Yeah, I'm not sure either :-), let me change this sentence slightly and drop the package name as it will be shown in the example below

Reply:
@michalvavrik I've kept the package name but restructured a bit, from what I've seen, omitting `the` seems acceptable when referring to (Java) class names, but it can be easily tuned if needed

Reply:
Thanks, read well now.